### PR TITLE
feat: add grafana datasource for psql

### LINKED ISSUE #14: Provision psql datasource in grafana at startup
Right now there are no datasources in the grafana even when we want to do metrics visualization from our Psql db.

### PR DESCRIPTION
**Pull request by sakshi1215**
fixes #14

### CODE REVIEW

**Review by jennydaman — CHANGES_REQUESTED** [review]
I don't think it makes sense to mix YAML and INI config syntax.

**Comment by sakshi1215** [comment]
I thought I had changed the ini format to yaml syntax but looks like I
missed pushing that commit, let me fix it.

**Comment by sakshi1215** [comment]
@jennydaman It is fixed now, please take a look. Ini & datasources are two different files which can't be merged, idk why I configured it in the helm-chart way where you can merge ini in your values.yaml.

File: docker-compose.yml [hunk 1]
```diff
@@ -284,6 +284,7 @@ services:
     volumes:
       - grafana_data:/var/lib/grafana
       - ./grafana.ini:/etc/grafana/grafana.ini:ro
+      - ./provisionaning:/etc/grafana/provisioning
```
Comment (jennydaman) [inline docker-compose.yml:287]: the dir name contains a typo, would you fix this?

**Review by jennydaman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**